cmd.go

```diff
@@ -153,6 +153,26 @@ func NewCmdOptions(options Options, name string, args ...string) *Cmd {
 	return out
 }
 
+// Clone clones a Cmd. All the options are transferred,
+// but the state of the original object is lost.
+// Cmd is one-use only, so if you need to re-start a Cmd,
+// you need to Clone it.
```

Comment:
Does the cloned Cmd reuses the original's `Stdout` and `Stderr` channels?

Reply:
Maybe I should document this another way, so it's more clear.

stdout, stderr and the status are dropped when cloning.
Only the public properties of the Cmd are transferred to the clone.
I'll add this ^ to the comment of the function.

Reply:
Oh wait, you mean the public Stdout and Stderr?
No, they shouldn't be transferred to the clone, because you want a **fresh** copy, ready to run from scratch. That's the point of the function.

Reply:
Ah yes, right: fresh `Stdout` and `Stderr`. 👍